test/images/volumes-tester/gluster/run_gluster.sh

```diff
@@ -14,17 +14,24 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+DIR=`mktemp -d`
+
 function start()
 {
+    mount -t tmpfs test $DIR
+    chmod 755 $DIR
+    cp /vol/* $DIR/
     /usr/sbin/glusterd -p /run/glusterd.pid
-    gluster volume create test_vol `hostname -i`:/vol force
+    gluster volume create test_vol `hostname -i`:$DIR force
     gluster volume start test_vol
 }
 
 function stop()
 {
     gluster --mode=script volume stop test_vol force
     kill $(cat /run/glusterd.pid)
+    umount $DIR
+    rm -rf $DIR
```

Comment:
Just want to double check whether we need to retry to kill/umount in case command fails? (Any scenarios these commands might fail?)